tests/testthat/helper-all.equal.R

```diff
@@ -1,2 +1,4 @@
-# covr package doesn't detect the .S3method if it is declared within R/ folder
-.S3method("all.equal", "join_keys", all.equal.join_keys)
+# Register method for testthat::expect_identical to compare join_keys
+if (nzchar(Sys.getenv("TESTTHAT"))) {
```

Comment:
have you looked into `testthat::is_testing()` (and possibly other functions like this) 
docs: https://testthat.r-lib.org/reference/is_testing.html

This looks very close to what this function does but the function might also account other edge cases

Reply:
(TIL) I wasn't aware of that function, that is a better solution!

079461a5